models.py

```diff
@@ -339,6 +388,9 @@ def is_accessible(self):
         if current_user.has_role('admin'):
             self.can_export = True
             return True
+        if current_user.has_role('merchant'):
```

Comment:
just make it so that all users can export

Reply:
remove self.can_export for the role. Added can_export on the function instead so all users can export for merchant tx tab, providing they can access it.

Reply:
removed user and nonce from the api_key

Reply:
just make can_export true for everyone